src/DynamoCore/Graph/NodeGraph.cs

```diff
@@ -33,6 +33,8 @@ public class NodeGraph
         /// </summary>
         public List<NoteModel> Notes { get; private set; }
 
+        public List<ConnectorPinModel> Pins { get; private set; }
```

Comment:
needs a summary

Reply:
Actually not sure that we even _need_ this property... It isnt being called anywhere, created it prematurely..

Reply:
Not sure what your thoughts are here. Seems like the xml is mostly used for undo/redo actions, not so much for loading workspaces from xml. Thus the maybe lack of need to reconstruct pins from the nodegraph...

Reply:
yea if we can get rid of it lets do that. xml should only be for undo/redo yea